### ISSUE TITLE
collector

### ISSUE THREAD

**Issue by DiLinTian** [wwfxuk/tk-katana#26]
**Describe the bug**

**To Reproduce**
tk-multi-publish2.yml:
settings.tk-multi-publish2.katana.asset_step:
  collector: "{self}/collector.py:{engine}/tk-multi-publish2/basic/collector.py"

Hello, thank you very much for your development and sharing. But I have a problem: After Configuring with your tk-katana, I find that the collector. py is missing. Where is this file? Or do I need to develop it myself?

**Comment by Liametc** [comment]
Hi @DiLinTian
 
This isn't our primary repo for tk-katana, so there is no current development on it. We have been working closely with the Foundry to develop the official release, which is currently private. You may be able to get access to that repo by messaging them and asking to have access or wait until they make it public. I can assure you _that_ one definitely contains the collector module.

Apologies for not being able to help you out more

**Comment by j0yu** [comment]
Yeah, we're currently working on the C++ plugin based `tk-katana` with the Foundry.

But you're right, I think the missing `{engine}/tk-multi-publish2/basic/collector.py` was from me accidentally deleting it while working on #22

For now, you can grab a copy from [the commit before that PR: d3a524](https://github.com/wwfxuk/tk-katana/blob/d3a524d7531cc71623c183581c928fb595b3560c/hooks/tk-multi-publish2/basic/collector.py)

**Comment by DiLinTian** [comment]
@Liametc @j0yu Thank you very much for your reply. I see, and I'm looking forward to the official release.
